stage1/init/pod.go

```diff
@@ -249,7 +249,7 @@ func (p *Pod) appToSystemd(ra *schema.RuntimeApp, am *schema.ImageManifest, inte
 		}
 	}
 
-	for _, i := range am.App.Isolators {
+	for _, i := range app.Isolators {
```

Comment:
@jonboulle I was suggesting that in https://github.com/coreos/rkt/pull/1002#issuecomment-110498256